Python_BasicLiterature_MitBeispiel.md

```diff
@@ -1,104 +1,159 @@
 # i can comment with pound sign . yes, that's also comment
 
 # Assigning string to a variable
+
 a = "This is a string"
+
 print(a)
-#The output is : This is string
+
+##The output is : This is string
```

Comment:
you need to add space after a headline 
in your case it should look like this
`## the output is: this is string`